### ISSUE TITLE
Question about Prob3 Test 1 Error: Unable to resolve the name referenceVariables.numTest.

### ISSUE THREAD

**Issue by CharlotteTheSpider** [chulminy/CIVE497-CIVE700#91]
I keep getting error message saying: 

    "Unable to resolve the name referenceVariables.numTest.
    Error in Test1"

I have run my code in matlab and have used assertion functions in my local matlab code to test if my results are the same as PokerHiden.p. However, I still keep getting the error message... I passed Test 2. 

Anyone has experienced the same thing? Any idea?

**Comment by chulminy** [comment]
This looks like there is no 'numTest' variable after running on your program. 

Maybe, you overwrite the variable... you should not delete or change 'numTest' variable because my test code refers to the variable. Or your code has an error (in this case, you first do 'run script'). 

Here is a part of my test code. The test code fails to read 'numTest'.

![image](https://user-images.githubusercontent.com/3227103/72558654-23204a00-3871-11ea-8bca-93f602f96b32.png)

**Comment by CharlotteTheSpider** [comment]
Yup.... I figured it out. I have copied everything in the matlab file so that overwrites something. I am good now! Thank you for your help.
